ProcessMaker/processmaker
# Add API Documentation for Task Assignmnets

## Issue & Reproduction Steps
TICKET: [FOUR-5983](https://processmaker.atlassian.net/browse/FOUR-5983)
As a process architect, I want to have documentation of the task assignments API, so I can manage task assignments

## Solution
- Add missing API documentation for Task Assignments

## How to Test

1. Navigate to `/api/documentation` in your local instance.
2. Authorize yourself via Bearer Token
3. Locate the 'Task Assignment' location
4.  'Try Out' all APIs
5. Ensure all fields work properly
6. Ensure all properties are listed in the example schema

## Related Tickets & Packages
NONE

## Code Review Checklist
